docs/frequency-analysis/frequency-analysis.md

```diff
@@ -0,0 +1,10 @@
+Frequency analysis can be used to count letters, but in our case,
+we are counting the frequency of meaningful words and phrases and filtering out stopwords.
```

Comment:
I think the general flow of the sentence might be improved if you simply say how we're using frequency analysis without mentioning other uses. Other things can be mentioned in the end of the paragraph if needed